eventarc/audit_iam/main.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2023 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START eventarc_audit_iam_server]
+import json
+import os
+
+from cloudevents.http import from_http
+from flask import Flask, request
+from google.events.cloud.audit import LogEntryData
+
+app = Flask(__name__)
+if __name__ == "__main__":
+    app.run(debug=True, host="0.0.0.0", port=int(os.environ.get("PORT", 8080)))
+
+# [END eventarc_audit_iam_server]
+
+
+# [START eventarc_audit_iam_handler]
+@app.route("/", methods=["POST"])
+def index():
+    # Transform the HTTP request into a CloudEvent
+    event = from_http(request.headers, request.get_data())
+
+    # Extract the LogEntryData from the CloudEvent
+    # re-serialize to json, to convert the json-style 'lowerCamelCase' names to the protobuf-style 'snake_case' equivalents.
+    # ignore_unknown_fields is needed to skip the '@type' fields.
+    log_entry = LogEntryData.from_json(
```

Comment:
Do we want to link to https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry?